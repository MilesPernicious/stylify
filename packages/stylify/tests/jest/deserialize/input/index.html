<div class="color:steelblue"></div>
