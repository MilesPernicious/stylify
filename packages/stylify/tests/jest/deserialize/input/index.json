{"mangleSelectors":false,"reconfigurable":true,"dev":true,"screensList":{"_":0,"(min-width: 640px)":1,"(min-width: 1024px) and (max-width: 1280)":2,"(min-width: 1280px), (orientation: landscape)":3},"selectorsList":{"minw640px:color:blue":{"screenId":1,"selector":"minw640px:color:blue","properties":{"color":"blue"},"mangledSelector":"s0"},"minw1024px&&maxw1280:color:green":{"screenId":2,"selector":"minw1024px&&maxw1280:color:green","properties":{"color":"green"},"mangledSelector":"s1"},"minw1280px||landscape:color:purple":{"screenId":3,"selector":"minw1280px||landscape:color:purple","properties":{"color":"purple"},"mangledSelector":"s2"},"minw1280px||landscape:hover:color:purple":{"screenId":3,"selector":"minw1280px||landscape:hover:color:purple","properties":{"color":"purple"},"mangledSelector":"s3","pseudoClasses":["hover"]},"color:red":{"screenId":0,"selector":"color:red","properties":{"color":"red"},"mangledSelector":"s4"}}}
