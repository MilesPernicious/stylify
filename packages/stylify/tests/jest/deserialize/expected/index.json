{"mangleSelectors":false,"reconfigurable":true,"dev":true,"selectorsList":{"color:red":{"mangledSelector":"s0","processed":true},"minw640px:color:blue":{"mangledSelector":"s1","processed":true},"minw1024px&&maxw1280:color:green":{"mangledSelector":"s2","processed":true},"minw1280px||landscape:color:purple":{"mangledSelector":"s3","processed":true},"minw1280px||landscape:hover:color:purple":{"mangledSelector":"s4","processed":true},"color:steelblue":{"mangledSelector":"s5","processed":true}},"cssTree":{"_":{"color:red":{"selectors":["color:red"],"properties":{"color":"red"}},"color:steelblue":{"selectors":["color:steelblue"],"properties":{"color":"steelblue"}}},"(min-width: 640px)":{"minw640px:color:blue":{"selectors":["minw640px:color:blue"],"properties":{"color":"blue"}}},"(min-width: 1024px) and (max-width: 1280)":{"minw1024px&&maxw1280:color:green":{"selectors":["minw1024px&&maxw1280:color:green"],"properties":{"color":"green"}}},"(min-width: 1280px), (orientation: landscape)":{"minw1280px||landscape:color:purple":{"selectors":["minw1280px||landscape:color:purple"],"properties":{"color":"purple"}},"minw1280px||landscape:hover:color:purple":{"selectors":["minw1280px||landscape:hover:color:purple:hover"],"properties":{"color":"purple"}}}}}